### PR TITLE
[bugfix] random.choice error with python3.11 (blur/transforms.py, ver…

### PR DESCRIPTION
**Pull request by whlook**
…sion on 1.3.0)

  err info:ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
  fix with array.tolist()

### CODE REVIEW

**Comment by Dipet** [comment]
Could you provide a reproducible example of the error?

**Comment by Dipet** [comment]
Related #1357

**Comment by Dipet** [comment]
Please change it inside `Blur` too.

File: albumentations/augmentations/blur/transforms.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def apply(self, img: np.ndarray, ksize: int = 3, **params) -> np.ndarray:
         return F.blur(img, ksize)
 
     def get_params(self) -> Dict[str, Any]:
-        return {"ksize": int(random.choice(np.arange(self.blur_limit[0], self.blur_limit[1] + 1, 2)))}
+        return {"ksize": int(random.choice(np.arange(self.blur_limit[0], self.blur_limit[1] + 1, 2).tolist()))}
```
Comment (Dipet) [inline albumentations/augmentations/blur/transforms.py:44]: use `range` instead of `np.arange`

**Comment by whlook** [comment]
> It looks like we need to replace all `np.arange` with a `range`

Have tried itSomething wrong with 'random.choice(np.arange...'，

File: albumentations/augmentations/blur/transforms.py (hunk 1 above)
Comment (whlook) [reply albumentations/augmentations/blur/transforms.py:44]: Have changed to 'range'. It seems a problem with the combination of 'random.choice' and np.arange,so i just fix 'arange' on these cases.

**Pull request closed without merging**